fwoa-tools/src/migrationImport.ts

```diff
@@ -296,6 +297,7 @@ async function checkConfiguration(): Promise<void> {
     } catch (e) {
       console.log('import failed!', e);
       logs.write(`\n**${new Date().toISOString()}: ERROR!**\n${e}\n`);
+      logs.end();
```

Comment:
We can move this outside of the `if statement` to line 305. That way we'll capture logs in line 291, 292, 293 as well as any logs in the `if` statement.

Reply:
Updated 👍